### DISCUSSION TITLE
Long-term support for rspack regardless the ongoing rumours about Tiktok ban in US

### DISCUSSION THREAD

**Discussion by zizizi17** [web-infra-dev/rspack#6856] [General]
Hey fols
Sorry, if this topic was raised already but did not find any reference to it. 

Would like to ask the team what is the long-term plans on rspack in terms on either becoming community supported or Tiktok? 
Especially taking into account ongoing news/rumours about plans to ban the Tiktok in some counties. 
Will it be backed, supported and developing regardless this? 
If to think about migrating from webpack to rspack in safe to do it?

**Comment by chenjiahan** [comment]
Thank you for your issue, Rspack is a ByteDance open source project, so generally it will not be affected by TikTok. If you are interested in TikTok's ongoing PR issues, you could contact <pr@tiktok.com> for further info, hope that will help.

The Rspack team will focus on building the webpack-compatible Rust bundler and its ecosystem, providing the long-term maintenance support. We are also working with more community developers and teams to allow more people to contribute to the Rspack ecosystem.

We look forward to helping more webpack projects migrate to Rspack and get a better development experience. If you encounter any issues, feel free to give us feedback via GitHub, Discord or Twitter.
